src/OrchardCore/OrchardCore.Abstractions/Shell/Extensions/ShellFeaturesManagerExtensions.cs

```diff
@@ -16,9 +18,21 @@ public static class ShellFeaturesManagerExtensions
             IEnumerable<IFeatureInfo> features, bool force)
         {
             var (_, featuresToEnable) = await shellFeaturesManager.UpdateFeaturesAsync([], features, force);
+
             return featuresToEnable;
         }
 
+        public static async Task EnableFeaturesAsync(this IShellFeaturesManager shellFeaturesManager, params string[] featureIds)
+        {
+            ArgumentNullException.ThrowIfNull(featureIds);
```

Comment:
You need to check if the array is empty too

Reply:
It would do nothing when array is empty

Reply:
IMHO know need to call `GetAvailableFeaturesAsync()` if the `featureIds` is empty

Reply:
@hishamco done. anything else?

Reply:
The second time you can request a review rather than mention